<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminCenterdialog</class>
 <widget class="QDialog" name="AdminCenterdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理中心</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>680</width>
     <height>560</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab1">
    <attribute name="title">
     <string>书籍管理</string>
    </attribute>
    <widget class="QTableView" name="Allbook">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>370</width>
       <height>511</height>
      </rect>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>389</x>
       <y>10</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>编号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineBookno"/>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSearch">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonDelete">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTableView" name="InsertBook">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>291</y>
       <width>271</width>
       <height>231</height>
      </rect>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>60</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>书籍入库</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>100</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>编号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="bookno"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>类别：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="category"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>140</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>标题：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="title"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>作者：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="author"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>180</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="d">
        <property name="text">
         <string>出版：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="publisher"/>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>价格：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="price"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>220</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>年份：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="year"/>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>数量：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="count"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="multi">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>260</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>多本入库</string>
     </property>
    </widget>
    <widget class="QPushButton" name="single">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>260</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>单本入库</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>260</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonConfirm">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>260</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab2">
    <attribute name="title">
     <string>用户管理</string>
    </attribute>
    <widget class="QTableView" name="user">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>321</width>
       <height>381</height>
      </rect>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTableView" name="theirbook">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>110</y>
       <width>321</width>
       <height>381</height>
      </rect>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>用户列表：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="detailsUser">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>500</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>查看详细</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteuser">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>500</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>删除用户</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="buttonReturn">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回主菜单</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
